source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -46,12 +46,19 @@ public void go() {
     private void nearExample() {
         // begin findExample
 
-        // code to set up your mongo client ...
+        // Add your MongoCollection setup code here
         MongoDatabase database = mongoClient.getDatabase("sample_mflix");
         MongoCollection<Document> collection = database.getCollection("theaters");
+
         Point centralPark = new Point(new Position(-73.9667, 40.78));
+        
+        // Creates a query that matches all theaters between 5,000 and 10,000 meters from the specified Point
```

Comment:
Issue:
By itself, this query does not match theaters. Instead, it just matches all points within a range from the starting coordinates.

Suggestion:
```suggestion
        // Creates a query that matches all locations between 5,000 and 10,000 meters from the specified Point
```